create-ladder.js

```diff
@@ -17,4 +16,15 @@ const unfoldQ = (elem,inx) =>{
   }
 }
 
-window.onload = showLadder;
+const addHandlersToElements = () => {
```

Comment:
Can we find a word better than `Elements`? its pretty vague, but I see the function is targeting something specific.